bin/protogen

```diff
@@ -62,6 +62,12 @@ def main(args=sys.argv[1:]):
             "consensus/poet/families/sawtooth_validator_registry/protos")
     protoc(proto_dir, "consensus/poet/common", "sawtooth_poet_common/protobuf")
 
+    # 4. Generate supplychain protos
+    proto_dir = JOIN(TOP_DIR, "families/supplychain_python/protos")
```

Comment:
It feels like the family name shouldn't contain the implementation language. The python specific implementation should be one level lower, and then the protos can be at the level that is language agnostic.